ui/pages/settings/info-tab/info-tab.component.js

```diff
@@ -53,6 +57,10 @@ export default class InfoTab extends PureComponent {
   componentDidMount() {
     const { t } = this.context;
     handleSettingsRefs(t, t('about'), this.settingsRefs);
+    if (!this.props.tesRemoteFeatureFlag) {
```

Comment:
Nit: typo, `tesRemoteFeatureFlag` should probably be `testRemoteFeatureFlag`

Reply:
addressed in 89fe7aced2e9557d1841598bead82c5ac897cfb9